tests/python/unittest/test_tir_usmp_analysis_extract_bufferinfo.py

```diff
@@ -1365,9 +1362,9 @@ def run_model(data: T.handle, output: T.handle) -> None:
 
 def test_multiple_calls_to_same_primfunc():
```

Comment:
We need a unit test here to check the newly added feature to extract buffer info pass.

Reply:
Added

Reply:
Was this added? I couldn't find it after a naive look